src/EnergyPlus/SimAirServingZones.hh

```diff
@@ -66,10 +66,6 @@ struct EnergyPlusData;
 
 namespace SimAirServingZones {
 
-    // coil operation
-    constexpr int CoilOn(1);  // normal coil operation
-    constexpr int CoilOff(0); // signal coil shouldn't run
```

Comment:
These `CoilOn` and `CoilOff` variables were replaced with `CompressorOperation::On` and `CompressorOperation::Off` respectively. Reason: the argument of `SimHXAssistedCoolingCoil()` where they are used signals compressor operation.